src/features/import/utils/prepareImportOperations.spec.ts

```diff
@@ -0,0 +1,790 @@
+import prepareImportOperations from './prepareImportOperations';
+import { ColumnKind, Sheet } from './types';
+import { describe, it } from '@jest/globals';
+
+describe('prepareImportOperations when first row is header', () => {
```

Comment:
I would wrap both this `describe` and the one on line 383 in an outer describe, like this:

```ts
describe('pareImportOperations()', () => {
  describe('when first row is header', () => {
    // it tests
  });

  describe('when first row is not header', () => {
    // it tests
  });
});
```